modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1090,8 +1090,10 @@
     <string name="settings_notification_choose_podcasts">Choose podcasts</string>
     <string name="settings_notification_hide_on_pause">Hide playback notification on pause</string>
     <string name="settings_notification_notify_me">Notify me</string>
-    <string name="settings_notification_play_over">Play over notifications</string>
-    <string name="settings_notification_play_over_summary">Keep playing even when other apps, like notifications or navigation, play sounds.</string>
+    <string name="settings_notification_play_over">Play over external notifications</string>
```

Comment:
I noticed that you added "external" to this string. Do you find "notifications" on it's own to be unclear? Just asking because I'm curious.

Reply:
I think when I worked on it pre vacation I must have thought so, after some time away and re-reading it though I'm happy to leave "external" out of it.